egs2/cmu_indic/tts1/sid

```diff
@@ -0,0 +1 @@
+../../../tools/kaldi/egs/sre08/v1/sid
```

Comment:
Is this part used somewhere?

Reply:
This is used to extract x-vector. Only used for multi-speaker case.
(This symlink is included in TEMPLATE.)